indexers/keyvalue/LevelDBIndexer/requirements.txt

```diff
@@ -1,2 +1,2 @@
-plyvel
-jina
\ No newline at end of file
+plyvel==1.3.0
+jina==0.8.2
```

Comment:
It is not really important because it is not used from here, but watch with this change because every jina version will have its own hub build. So I would not fix the version

Reply:
Interesting. Okay, I will remove the version number on the jina dependency. How did it work before with the most recent jina version. I had to update the Document type to let it run with v0.8.2
Since every jina version will have its own hub build, how was it even possible to build the LevelDBIndexer for the latest jina version?

Reply:
It did not succeed, that is why this issue was automatically generated https://github.com/jina-ai/jina-hub/issues/870